src/main/kotlin/no/nav/syfo/infrastructure/mq/MQUtils.kt

```diff
@@ -17,7 +17,6 @@ fun connectionFactory(env: MQEnvironment): javax.jms.ConnectionFactory =
         setIntProperty(CommonConstants.WMQ_PORT, env.mqPort)
         setStringProperty(CommonConstants.WMQ_CHANNEL, env.mqChannelName)
         setIntProperty(CommonConstants.WMQ_CCSID, UTF_8_WITH_PUA)
-        setStringProperty(CommonConstants.WMQ_SSL_CIPHER_SUITE, "*TLS13ORHIGHER")
```

Comment:
Denne skal brukes hvis man vil ha en kryptert forbindelse mot MQ. Vi bruker det i noen andre sammenhenger (feks lesing av dialogmeldinger), men den er litt ekstra mekk med det - så tenkte å ikke ha en kryptert forbindelse for denne app'en.